code/__HELPERS/time.dm

```diff
@@ -20,6 +20,10 @@
 /proc/SQLtime(timevar)
 	return time2text(timevar || world.timeofday, "YYYY-MM-DD hh:mm:ss")
 
+/proc/station_time(display_only = FALSE, wtime=world.time)
+
+/proc/station_time_timestamp(format = "hh:mm:ss", wtime)
+	return time2text(station_time(TRUE, wtime), format)
```

Comment:
These two procs can be removed after following my other suggestions in this review.